vitest.setup.simple.ts

```
@@ -72,3 +278,76 @@ Object.defineProperty(window, "location", {
+    select: vi.fn(() => createChainableMock(table, "select")),
+    insert: vi.fn(() => createChainableMock(table, "insert")),
+    update: vi.fn(() => createChainableMock(table, "update")),
+    delete: vi.fn(() => createChainableMock(table, "delete")),
+    eq: vi.fn((column, value) => {
+      // Handle specific test cases
+      if (table === "ai_sessions" && value === "test-session-fail") {
+        return {
+          ...createChainableMock(table, operation),
+          eq: vi.fn(() => createChainableMock(table, operation)),
+          then: vi.fn((resolve) =>
+            resolve({ data: null, error: { message: "Session not found" } })
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)